### ISSUE TITLE
ctrl key acts as fn key?

### ISSUE THREAD

**Issue by trusktr** [mbaltaks/doublecommand#22]
I was looking at the screen shot, and I didn't see an option to make the ctrl key act like the fn key? Is there any such feature? I'd like to ultimately swap my fn and ctrl keys.

**Comment by oschrenk** [comment]
:+1: That would be amazing for my external keyboard

**Comment by aerohit** [comment]
Yeah, I also would like to have that feature for my external keyboard.

**Comment by 7fe** [comment]
Yes I agree
